app/controllers/spaces_controller.rb

```diff
@@ -14,18 +14,23 @@ def create
     space_type = SpaceType.find_or_create_by!(type_name: 'Skole')
     @space = Space.create!(space_type: space_type, **space_params)
 
-    redirect_to spaces_path
+    redirect_to space_path(@space)
   end
```

Comment:
I think this makes more sense in the future, while redirecting to the index was useful before we had a proper space to show.